src/auth/models/strategy.class.ts

```diff
@@ -80,16 +80,14 @@ export abstract class Strategy extends events.EventEmitter {
             this.logger.log('logout start')
             const { accessToken, refreshToken } = req.session?.passport.user.tokenset
 
-            const auth = `Basic ${Buffer.from(`${this.options.clientID}:${this.options.clientSecret}`).toString(
-                'base64',
-            )}`
+            const auth = this.getAuthorization(this.options.clientID, this.options.clientSecret)
 
-            await http.delete(`${this.options.logoutURL}/session/${accessToken}`, {
+            await http.delete(this.urlFromToken(this.options.logoutURL, accessToken), {
```

Comment:
how about we just move this method? e.g. ```deleteTokens(tokens: string[])```?

Reply:
@paddybasi  will do in next PR. Thanks